internal/serviceapi/alertconfigurationapi/resource_test.go

```diff
@@ -0,0 +1,748 @@
+			},
+		},
+	})
+}
+
+func TestAccAlertConfigurationAPI_withOpsGenie(t *testing.T) {
+	var (
+		projectID = acc.ProjectIDExecution(t)
+		apiKey    = dummy36CharKey
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             checkDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: configWithOpsGenie(projectID, apiKey, true),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					checkExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "group_id", projectID),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAlertConfigurationAPI_withVictorOps(t *testing.T) {
+	var (
+		projectID = acc.ProjectIDExecution(t)
+		apiKey    = dummy36CharKey
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             checkDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: configWithVictorOps(projectID, apiKey, true),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					checkExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "group_id", projectID),
+				),
+			},
+		},
+	})
+}
+
+func TestAccAlertConfigurationAPI_withSeverityOverride(t *testing.T) {
+	var (
+		projectID = acc.ProjectIDExecution(t)
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:                 func() { acc.PreCheckBasic(t) },
+		ProtoV6ProviderFactories: acc.TestAccProviderV6Factories,
+		CheckDestroy:             checkDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: configWithSeverityOverride(projectID, conversion.StringPtr("ERROR")),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					checkExists(resourceName),
+					resource.TestCheckResourceAttr(resourceName, "severity_override", "ERROR"),
+				),
+			},
+			// TODO: Should check for no attr once CLOUDP-353933 is fixed.
+			// {
+			// 	Config: configWithSeverityOverride(projectID, nil),
+			// 	Check: resource.ComposeAggregateTestCheckFunc(
+			// 		checkExists(resourceName),
+			// 		resource.TestCheckNoResourceAttr(resourceName, "severity_override"),
+			// 	),
+			// },
+		},
+	})
+}
+
+func checkExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("not found: %s", resourceName)
+		}
+		id := rs.Primary.Attributes["id"]
+		projectID := rs.Primary.Attributes["group_id"]
+		if id == "" || projectID == "" {
+			return fmt.Errorf("checkExists, attributes not found for: %s", resourceName)
+		}
+		_, _, err := acc.ConnV2().AlertConfigurationsApi.GetAlertConfig(context.Background(), projectID, id).Execute()
+		if err != nil {
+			return fmt.Errorf("the Alert Configuration(%s) does not exist", id)
+		}
```

Comment:
```suggestion
		if _, _, err := acc.ConnV2().AlertConfigurationsApi.GetAlertConfig(context.Background(), projectID, id).Execute(); err != nil {
			return fmt.Errorf("the Alert Configuration(%s) does not exist", id)
		}
```

nit, err scope is restricted to this block

Reply:
👌 applied [006b01d](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/3880/commits/006b01d7f05204f8793c8f67e92fe71b393349e4)